core/manifest/ConfigStaticManifest.php

```diff
@@ -199,6 +199,25 @@ protected function next($ignoreWhitespace = true) {
 	}
 
 	/**
+	 * Get the previous token processed. Does *not* decrement the pointer
+	 *
+	 * @param bool $ignoreWhitespace - if true will skip any whitespace tokens & only return non-whitespace ones
+	 * @return null | mixed - Either the previous token or null if there isn't one
+	 */
+	protected function lastToken($ignoreWhitespace = true) {
+		// Subtract 1 as the pointer is always 1 place ahead of the current token
+		$pos = $this->pos - 1;
+		do {
+			if($pos <= 0) return null;
+			$pos--;
+			$prev = $this->tokens[$pos];
```

Comment:
Decrementing `$pos` separately before inspecting the `$this->tokens` array for the same reason highlighted above